internal/pkg/state/load_local_test.go

```diff
@@ -265,7 +265,9 @@ func loadLocalTestState(t *testing.T, m *manifest.Manifest, fs filesystem.Fs) (*
 	// Load state
 	project, err := d.Project()
 	assert.NoError(t, err)
-	state, _, localErr, remoteErr := LoadState(project, Options{LoadLocalState: true}, d)
+	state, err := New(project, d)
+	assert.NoError(t, err)
+	_, localErr, remoteErr := state.Load(Options{LoadLocalState: true})
```

Comment:
Rozdelena funkcia `LoadState` na `New` a metodu `state.Load`.
Kedze chyby pri `New` su fatalne a ukoncujeme beh.
Ale niektore chyby pri `Load` ignorujeme.
Takto je to cistejsie, a viem v teste jednoducho vytvort `empty state`.